extension/js/common/core/common.ts

```diff
@@ -312,3 +312,7 @@ export class Url {
   }
 
 }
+
+export const emailKeyIndex = (scope: string, key: string): string => {
```

Comment:
Actually, even for the in memory implementation could use `singleScopeRawIndex` then this would not be needed to be separately here. If you'd like this method to be separate, I'd prefer that it's placed in AbstractStore if possible?

Reply:
AbstractStore is platform-dependent. It won't be possible to write a simple unit-node test, only browser tests